packages/swap/src/routes/swap.ts

```diff
@@ -177,20 +162,21 @@ router.get(
           name: failedSwap.reason,
           message: failedSwapMessage[failedSwap.reason],
         };
-      } else if (swapRequest?.ignoredEgress) {
-        switch (swapRequest.ignoredEgress.type) {
+      } else if (hasIgnoredEgresses) {
+        const ignored = swapRequest?.ignoredEgresses.at(0);
+        switch (ignored!.type) {
           case 'REFUND':
             failureMode = Failure.RefundEgressIgnored;
```

Comment:
I feel like it's better to consider a REFUNDED as a Completed swap.
We can keep Failed only for the cases when something went wrong and the money was not received by the sender or the receiver.
What do you think?

Reply:
thats only the case when the refund egress is ignored.. Refunded is still a completed swap

Reply:
ah correct. you're right
Thanks!